Library/Formula/legit.rb

```diff
@@ -55,6 +56,10 @@ def install
 
     bin.install Dir["#{libexec}/bin/*"]
     bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"])
+
+    if build.head?
+      man1.install "extra/man/legit.1"
+    end
```

Comment:
You can inline this: `man1.install "extra/man/legit.1" if build.head?`. Why is it done only when building `HEAD`?